src/less/servicesMenu.less

```diff
@@ -22,51 +21,73 @@
     height:100%;
     top:0px;
     left:0px;
-    z-index:900;
+    z-index: @zindex-services-menu - 1;
 
     &.hidden {
         display: none;
     }
 }
 
 #navbar-services {
+    height: @navbar-height;
     display: flex;
     flex-grow: 1;
+    position: relative;
+    padding-left: 11px;
+
+    &:before {
+      content: "";
+      position: absolute;
+      width: 1px;
+      top: 15px;
+      bottom: 15px;
+      left: 0px;
+      background-color: @gray-75;
+    }
 
     .navbar-services-menu {
-        font-family: "Open Sans", "Helvetica Neue", "Helvetica", Helvetica, Arial, sans-serif;
         display: flex;
         flex-direction: column;
         height: 45px;
         text-decoration: none;
-        border-radius: 5px;
-        padding: 3px 8px 8px 8px;
-        margin-top: 3px;
+        border-radius: 3px;
+        padding: 3px 8px 7px 8px;
```

Comment:
@jsmecham Can we adjust the padding here or on the arrow to ensure both sides are 10px?
![image](https://cloud.githubusercontent.com/assets/130654/25592969/b2017cc6-2e6f-11e7-8c1d-3eb6ae778a20.png)